README.md

```diff
@@ -0,0 +1,107 @@
+# Canvas-image-cover [![Build Status](https://secure.travis-ci.org/noblesamurai/canvas-image-cover.png?branch=master)](http://travis-ci.org/noblesamurai/canvas-image-cover) [![NPM version](https://badge-me.herokuapp.com/api/npm/canvas-image-cover.png)](http://badges.enytc.com/for/npm/canvas-image-cover)
+
+> Cover a canvas with an image given a locked centre point and zoom factor.
+
+## Usage
+
+```js
+const cover = require('canvas-cover-image');
+const Canvas = require('canvas');
+const Image = Canvas.Image;
+const canvas = new Canvas(200, 200);
+const ctx = canvas.getContext('2d');
+const img;
+
+// Assume image is 200x200px
+fs.readFile(__dirname + '/images/squid.png', function(err, squid){
+  if (err) throw err;
+  img = new Image;
+  img.src = squid;
+  overlay(img);
+  zoom(img);
+  offset(img);
+});
+
+// Perfect overlay of image to canvas
+function overlay (img) {
+  cover(ctx, img, 0, 0, 200, 200);
+}
+
+// Overlay image (at 2X zoom) to canvas.
+function zoom (img) {
+  cover(ctx, img, 0, 0, 200, 200, { zoom: 2 });
+}
+
+// Overlay image, and zoom with 0, 0 locked as centre point of zoom.
+function offset(img) {
+  cover(ctx, img, 0, 0, 200, 200, { zoom: 2, cx: 0, cy: 0 });
+}
+```
+
+## API
+<a name="cover"></a>
```

Comment:
do we need this? do we link to #cover anywhere? ... I believe github should automatically make #api links about here anyway?